src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -73,7 +73,7 @@
                 (styles/sheet insets window-height override-theme shell?))}
 
        (when shell?
-         [blur/view
+         [blur/ios-view
```

Comment:
Do we have issues for blur on Android here? Asking so I can try to fix it here #15953

Reply:
Yeah, with `blur/view` it looks incorrect on Android

Reply:
Thanks, I'll try to fix it on my PR 👍